src/Console/ChromeDriverCommand.php

```diff
@@ -217,26 +216,34 @@ protected function download($version, $os)
      * @param  string  $archive
```

Comment:
☝️ docblock needs to updated

Reply:
Thanks! Updated the extract docblock.

Reply:
The other thing I noticed is elsewhere in Dusk they state `@throws \Exception`.